superset/models/helpers.py

```diff
@@ -1873,9 +1874,9 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
                     if (
                         op
                         not in {
-                            utils.FilterOperator.EQUALS.value,
```

Comment:
I'm confused as this looks properly indented in `master` RN...    
https://github.com/apache/superset/blob/master/superset/models/helpers.py#L1875-L1878 (?)

Reply:
I updated using ruff and formated correctly